ee/clickhouse/models/property.py

```diff
@@ -534,26 +534,36 @@ def box_value(value: Any, remove_spaces=False) -> List[Any]:
 
 
 def get_property_values_for_key(key: str, team: Team, value: Optional[str] = None):
+    property_field, _ = get_property_string_expr("event", key, "%(key)s", "properties")
```

Comment:
Type checker failing here.

Not sure why tests aren't catching this bug though - worth taking a look at coverage?

You might want to take a look at `@test_with_materialized_columns` decorator for testing this.

Reply:
Good enough for now! :)